app/schemas/temperature.py

```diff
@@ -0,0 +1,20 @@
+from datetime import datetime
+
+from pydantic import BaseModel
+
+
+class TemperatureBase(BaseModel):
+    temperature: float
+    date_time: datetime

```

Comment:
The `date_time` field in `TemperatureBase` should have a default value or be marked as optional to align with the database model where `date_time` has a default value. Consider using `Optional[datetime]` with a default value.